Slim.js

```diff
@@ -956,8 +956,12 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
 
   // bind:property
   Slim.customDirective(function (attr) {
-    return (/^(bind):(\S*)/.exec(attr.nodeName)
-    );
+    if(attr.nodeName.indexOf("bind:") == 0) {
+      return attr.nodeName;
+    }    
+    else {
+      return null;
+    } 
   }, function (source, target, attribute, match) {
     var tAttr = match[2];
```

Comment:
`match` argument would be `bind:<something>` and therefore match[2] would be 'a'.

Did you run npm test before submitting?